custom/icds_reports/tests/reports/test_prevalence_of_severe.py

```diff
@@ -20,42 +20,48 @@ def test_map_data(self):
             )[0],
             {
                 "rightLegend": {
-                    "info": "Percentage of children between 0-5 years enrolled for ICDS services"
-                            " with weight-for-age less than -2 standard deviations "
-                            "of the WHO Child Growth Standards median. <br/><br/>"
-                            "Children who are moderately or severely underweight have a higher risk of mortality",
-                    "average": 17
+                    "info": "Percentage of children between 6 - 60 months enrolled for ICDS services with "
+                            "weight-for-height below -2 standard deviations of the WHO Child Growth Standards "
+                            "median. <br/><br/>Wasting in children is a symptom of acute undernutrition "
+                            "usually as a consequence of insufficient food intake or a high incidence "
+                            "of infectious diseases. Severe Acute Malnutrition (SAM) is nutritional "
+                            "status for a child who has severe wasting (weight-for-height) below -3 "
+                            "Z and Moderate Acute Malnutrition (MAM) is nutritional status for a child "
+                            "that has moderate wasting (weight-for-height) below -2Z.",
+                    "average": "1.05"
                 },
                 "fills": {
-                    "0%-20%": "#fee0d2",
-                    "20%-35%": "#fc9272",
-                    "35%-100%": "#de2d26",
+                    "0%-5%": "#fee0d2",
+                    "5%-7%": "#fc9272",
+                    "7%-100%": "#de2d26",
                     "defaultFill": "#9D9D9D"
                 },
                 "data": {
                     "st1": {
-                        "total": 475,
-                        "severely_underweight": 8,
-                        "moderately_underweight": 90,
-                        "fillKey": "20%-35%",
-                        "normal": 364
+                        "severe": 0,
+                        "moderate": 4,
+                        "normal": 5,
+                        "total_measured": 7,
+                        "total": 454,
+                        "fillKey": "0%-5%"
                     },
                     "st2": {
-                        "total": 514,
-                        "severely_underweight": 14,
-                        "moderately_underweight": 84,
-                        "fillKey": "0%-20%",
-                        "normal": 395
+                        "severe": 2,
+                        "moderate": 4,
+                        "normal": 16,
+                        "total_measured": 25,
+                        "total": 497,
+                        "fillKey": "0%-5%"
                     }
                 },
-                "slug": "moderately_underweight",
-                "label": "Percent of Children Underweight (0-5 years)"
```

Comment:
Why is this test changing? I don't see the complimentary change in the real code?

Reply:
@emord This test used the wrong data. As you can notice in the whole test we had prevalence_of_undernutrition data, but the test is for severe.

Reply:
Ah. I see that now. thanks